packages/create-spectacle/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "create-spectacle",
   "version": "0.1.0",
   "description": "Project generator for Spectacle",
-  "main": "bin/cli.js",
+  "main": "bin/src/cli.js",
```

Comment:
This should be `bin/cli.js`. See https://unpkg.com/browse/create-spectacle@0.1.0/bin/

Reply:
> Since the `package.json` file is used, the new TS root dir becomes `.` instead of `src`, so the bin location is now nested.